docs/proposal/1-authentication-support/README.md

```diff
@@ -0,0 +1,175 @@
+# kstone-api Supports Authentication
+
+## Motivation
+
+Kstone has become a popular solution for etcd management. Due to lack of authentication ability, it has safety vulnerability and is not recommended using in production environment. To improve safety features of kstone, we'd like to support authentication in kstone.
+
+## Goals
+
+1. Use username/password authentication to implement login feature(create an admin user by default). 
```

Comment:
1.  Designed to support multiple authentication plugins.
2. Strong security (password encryption, token expiration time, password reset only, etc.).
2. Built-in username/password authentication, the default is based on bearer token authentication (jwt).